tapir/accounts/templates/accounts/user_detail.html

```diff
@@ -25,6 +25,10 @@
                 <h5 class="card-header d-flex justify-content-between align-items-center flex-wrap gap-2">
                     <span>{% translate "Personal Data" %}</span>
                     <span class="d-flex justify-content-end flex-fill flex-wrap gap-2">
+                        <a class="{% tapir_button_link_to_action %}"
```

Comment:
This doesn't look very clean, especially for member-office users. Maybe as a side-menu item?

![image](https://github.com/user-attachments/assets/9f2d4875-fc87-45a7-bc5b-83a0438baec9)

Reply:
I don't think the UX is clean at all currently: You have "Gruppen bearbeiten", "Benutzername bearbeiten" and then an ominous general "Bearbeiten". I propose we continue with that menu for now, move all these other "edit" tools to a subpage in another PR and only keep "Bearbeiten" here